joy_node:
  ros__parameters:
    device_id: 0
    deadzone: 0.05
    autorepeat_rate: 20.0
    joy_config: "xbox"

teleop_node:
  ros__parameters:
    
    axis_linear:  # Left thumb stick vertical
      x: 1
    scale_linear:
      x: 0.5
    scale_linear_turbo:
      x: 1.0

    axis_angular:  # Left thumb stick horizontal
      yaw: 0
    scale_angular:
      yaw: 0.5
    scale_angular_turbo:
      yaw: 1.0

    gripper_topic: "/go"
    require_enable_button: true
    enable_button: 6  # Left shoulder button
    enable_turbo_button: 7  # Right shoulder button

joystick_operator_node:
    ros__parameters:
        deadman_button: 6 # Left shoulder
        slow_button: 7 # Right shoulder
        gripper_open_button: 0
        gripper_close_button: 1
        axis_z: 3
        vel_topic: "/arm_base_controller/commands"
        gripper_topic: "/gripper_controller/commands"
